src/components/tokens/TokenAction.js

```diff
@@ -134,6 +134,7 @@ class TokenAction extends React.Component {
     return (
       <div key={this.props.action}>
         <h2>{this.props.title}</h2>
+        <p>{this.props.subtitle}</p>
```

Comment:
This will add a blank height when there is no `subtitle`. `<p>` tags have `margin-bottom`.

Reply:
This is not a problem due to margin collapsing. `<p> margin-bottom` is 'ignored'.

`<p>`
![image](https://user-images.githubusercontent.com/3236756/63047207-ddc7ad00-becb-11e9-8393-fa903794c721.png)
`<form>`
![image](https://user-images.githubusercontent.com/3236756/63047237-f1731380-becb-11e9-847c-f73310ec3ffe.png)